pom.xml

```diff
@@ -192,19 +195,19 @@
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
-            <version>1.2.3</version>
+            <version>${logback.version}</version>

```

Comment:
Thanks. I really like using a property here since there's several dependencies with the same version.